### PR TITLE
Adding holding message to funding pages

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/gAOMPYPa/5053-add-holding-message-to-scholarships-and-bursaries-page

### Context

We should make it clear on the website that current funding amounts apply to the 2023/24 academic year.

### CODE REVIEW

**Comment by EllieNodder** [comment]
Approval given by Chris Cuff

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ navigation_description: Learn more about teacher training fees in England and fi
 date: "2021-05-27"
 image: "static/content/hero-images/0002.jpg"
 backlink: "../../"
+inset_text:
+  funding-holding-message:
+    text: |-
+      The following amounts apply if you're starting your teacher training course between September 2023 and July 2024. Financial support for courses starting the following year will be announced in autumn 2023.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md:21]: @EllieNodder Is this slightly confusing - the page talks about IRP, fees, loans etc. as well as bursaries and scholarships?

This banner implies that they are all being updated in autumn 2023 which I don't think is the case?

Should we be a bit more explicit?

```suggestion
      The bursary and scholarship amounts on this page apply if you're starting your teacher training course between September 2023 and July 2024. Financial support for courses starting the following year will be announced in autumn 2023.
```

**Review by philspray — APPROVED** [review]
Looks good to me.

File: app/views/content/subjects/engineers-teach-physics/_article.html.erb [hunk 2]
```diff
@@ -126,7 +126,7 @@
     <h2 class="heading-s">Funding for teaching physics</h2>
 
     <p>
-      Scholarships of £29,000 or bursaries of £27,000 are available for trainee physics teachers starting their teacher training course in the 2023/24 academic year.
+      Scholarships of £29,000 or bursaries of £27,000 are available for trainee physics teachers starting their teacher training course between September 2023 and July 2024.
```
Comment (gemmadallmandfe) [inline app/views/content/subjects/engineers-teach-physics/_article.html.erb:129]: I think this sounds a bit like if you start it in Sept 2024, you definitely won't get anything?

Last year we also added this - do we need something similar?
`Financial support for courses starting in 2023 will be announced in autumn 2022.`

(see https://github.com/DFE-Digital/get-into-teaching-app/pull/3559)

File: app/views/content/subjects/physics/_article.html.erb [hunk 3]
```diff
@@ -134,7 +134,7 @@
   <section class="col col-space-m col-720">
     <h3>Fund your training to teach physics</h3>
     <p>
-      Scholarships of £29,000 are available from the Institute of Physics. Alternatively, bursaries of £27,000 are available for trainee physics teachers starting their teacher training course from September 2023.
+      You could receive a scholarship of £29,000 from the Institute of Physics or a bursary of £27,000, if you start your teacher training course between September 2023 and July 2024.
```
Comment (gemmadallmandfe) [inline app/views/content/subjects/physics/_article.html.erb:137]: I think this sounds a bit like if you start it in Sept 2024, you definitely won't get anything?

Last year we also added this - do we need something similar?
Financial support for courses starting in 2023 will be announced in autumn 2022.

(see https://github.com/DFE-Digital/get-into-teaching-app/pull/2837)

File: app/views/content/subjects/physics/_article.html.erb [hunk 4]
```diff
@@ -153,7 +153,7 @@
 </p>
 
 <p>
-  You could also be eligible for a scholarship of £29,000 or a bursary of £27,000.
+  You could also be eligible for a scholarship of £29,000 or a bursary of £27,000 if you start your teacher training between September 2023 and July 2024.
```
Comment (gemmadallmandfe) [inline app/views/content/subjects/physics/_article.html.erb:156]: I think this sounds a bit like if you start it in Sept 2024, you definitely won't get anything?

Last year we also added this - do we need something similar?
Financial support for courses starting in 2023 will be announced in autumn 2022.

(see https://github.com/DFE-Digital/get-into-teaching-app/pull/2837)

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged first thing on Tuesday

**Pull request merged**